packages/custom-functions-metadata/test/cases/error-linkedentitydataprovider-excludefromautocomplete/functions.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+/**
+ * Test the linkedEntityDataProvider tag in combination with the excludeFromAutoComplete tag
+ * @param {any} linkedEntityId Unique `LinkedEntityId` of the `LinkedEntityCellValue`s which is being requested for resolution/refresh.
+ * @customfunction
+ * @linkedEntityDataProvider
+ * @excludeFromAutoComplete
+ * @returns {Promise<any>} Resolved/Updated `LinkedEntityCellValue` that was requested by the passed-in id.
+ */
+async function linkedEntityDataProviderTest(linkedEntityId: any): Promise<any> {
```

Comment:
Use `unknown` instead of `any` keyword.

Reply:
This is general guidance now for TypeScript where you don't have a specific known type. It allows code using the parameter to still have type info checked when narrowing the type, e.g. `if (typeof linkedEntityId === "string") { // type is narrowed to a string and will still be checked }`

Reply:
Please update the other places. I won't keep commenting on it.

Reply:
Hmm, using `unknown` instead of `any` results in this error: `Type doesn't match mappings typescript`

Reply:
Perhaps this PR should be completed using "any" and then another PR could make the changes to ensure `unknown` keyword is supported rather than just `any`.

Reply:
I think it's as simple as adding `[ts.SyntaxKind.UnknownKeyword]: "any"` to both TYPE_MAPPINGS_SIMPLE and TYPE_MAPPINGS.

Reply:
Ah yup, that did the trick! Thank you!